### PR TITLE
add my profile anassbouchfar

### PR DESCRIPTION
**Pull request by anassbouchfar**

### CODE REVIEW

File: public/data/anassbouchfar.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      {
+        "name": "E-mail",
+        "url": "mailto:anass.bouchfar22@gmail.com",
+        "icon": "envelope"
+      }
+      
+    ]
+    
+  }
+
```
Comment (Panquesito7) [inline public/data/anassbouchfar.json:35]:
```suggestion
}
```

File: public/data/anassbouchfar.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      },
+      {
+        "name": "Blog",
+        "url": "https://medium.com/@anass.bouchfar22",
+        "icon": ""
+      },
+      {
+        "name": "Portfolio",
+        "url": "https://anassbouchfar.me",
+        "icon": ""
```
Comment (Panquesito7) [inline public/data/anassbouchfar.json:24]:
```suggestion
        "icon": "website"
```

File: public/data/anassbouchfar.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
+{
+    "name": "ANASS BOUCHFAR",
+    "bio": "Web Developer , BI & Analytics Engineer Student",
```
Comment (Panquesito7) [inline public/data/anassbouchfar.json:3]:
```suggestion
    "bio": "Web Developer, BI & Analytics Engineer Student",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/anassbouchfar.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        "icon": "website"
+      },
+      {
+        "name": "E-mail",
+        "url": "mailto:anass.bouchfar22@gmail.com",
+        "icon": "envelope"
+      }
+      
+    ]
+
```
Comment (rohan-kulkarni-25) [inline public/data/anassbouchfar.json:33]: - Please delete these empty line

**Review by rohan-kulkarni-25 — CHANGES_REQUESTED** [review]

File: public/data/anassbouchfar.json (hunk 4 above)
Comment (krishguptadev) [inline public/data/anassbouchfar.json:33]:
```suggestion
```

File: public/data/anassbouchfar.json [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+      },
+      {
+        "name": "E-mail",
+        "url": "mailto:anass.bouchfar22@gmail.com",
+        "icon": "envelope"
+      }
+      
+    ]
+    
+}
```
Comment (krishguptadev) [inline public/data/anassbouchfar.json:34]:
```suggestion
}

```

File: public/data/anassbouchfar.json [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+        "name": "Portfolio",
+        "url": "https://anassbouchfar.me",
+        "icon": "website"
+      },
+      {
+        "name": "E-mail",
+        "url": "mailto:anass.bouchfar22@gmail.com",
+        "icon": "envelope"
+      }
+
```
Comment (krishguptadev) [inline public/data/anassbouchfar.json:31]:
```suggestion
```

File: public/data/anassbouchfar.json [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+        "url": "https://anassbouchfar.me",
+        "icon": "website"
+      },
+      {
+        "name": "E-mail",
+        "url": "mailto:anass.bouchfar22@gmail.com",
+        "icon": "envelope"
+      }
+      
+    ]
```
Comment (krishguptadev) [inline public/data/anassbouchfar.json:32]:
```suggestion
  ]
```

**Review by krishguptadev — CHANGES_REQUESTED** [review]

File: public/data/anassbouchfar.json [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+      },
+      {
+        "name": "Twitter",
+        "url": "https://twitter.com/anassbouchfar2",
+        "icon": "twitter"
+      },
+      {
+        "name": "Blog",
+        "url": "https://medium.com/@anass.bouchfar22",
+        "icon": ""
```
Comment (krishguptadev) [inline public/data/anassbouchfar.json:19]: Please add a icon here
```suggestion
        "icon": "book"
```

File: public/data/anassbouchfar.json [hunk 9]
```diff
@@ -0,0 +1,32 @@
+{
+    "name": "ANASS BOUCHFAR",
+    "bio": "Web Developer, BI & Analytics Engineer Student",
+    "avatar": "https://github.com/anassbouchfar.png",
+    "links": [
+      {
+        "name": "GitHub",
```
Comment (krishguptadev) [inline public/data/anassbouchfar.json:7]:
```suggestion
        "name": "Checkout my GitHub",
```

File: public/data/anassbouchfar.json [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+        "url": "https://github.com/anassbouchfar",
+        "icon": "github"
+      },
+      {
+        "name": "Twitter",
```
Comment (krishguptadev) [inline public/data/anassbouchfar.json:12]:
```suggestion
        "name": "Follow me on Twitter",
```

File: public/data/anassbouchfar.json [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+        "url": "https://twitter.com/anassbouchfar2",
+        "icon": "twitter"
+      },
+      {
+        "name": "Blog",
```
Comment (krishguptadev) [inline public/data/anassbouchfar.json:17]:
```suggestion
        "name": "Read my Blog",
```

File: public/data/anassbouchfar.json [hunk 12]
```diff
@@ -0,0 +1,32 @@
...
+        "url": "https://medium.com/@anass.bouchfar22",
+        "icon": ""
+      },
+      {
+        "name": "Portfolio",
```
Comment (krishguptadev) [inline public/data/anassbouchfar.json:22]:
```suggestion
        "name": "Visit my Website",
```

File: public/data/anassbouchfar.json [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+        "url": "https://anassbouchfar.me",
+        "icon": "website"
+      },
+      {
+        "name": "E-mail",
```
Comment (krishguptadev) [inline public/data/anassbouchfar.json:27]:
```suggestion
        "name": "Contact me via E-mail",
```

**Review by krishguptadev — CHANGES_REQUESTED** [review]

**Review by krishguptadev — APPROVED** [review]

**Review by schmelto — APPROVED** [review]

**Pull request merged**